docs/using.rst

```diff
@@ -93,10 +93,10 @@ s3front_    Y    Y    Integration with Amazon CloudFront distribution of S3 buck
 gandi_      Y    Y    Integration with Gandi's hosting products and API
 =========== ==== ==== ===============================================================
 
-.. _plesk: https://github.com/plesk/letsencrypt-plesk
-.. _haproxy: https://code.greenhost.net/open/letsencrypt-haproxy
-.. _s3front: https://github.com/dlapiduz/letsencrypt-s3front
-.. _gandi: https://github.com/Gandi/letsencrypt-gandi
+.. _plesk: https://github.com/plesk/certbot-plesk
+.. _haproxy: https://code.greenhost.net/open/certbot-haproxy
+.. _s3front: https://github.com/dlapiduz/certbot-s3front
+.. _gandi: https://github.com/Gandi/certbot-gandi
```

Comment:
We should not change any of these URLs. We do not know when/if they'll be renamed or what they'll be renamed to.